modules/mobile/spec/support/vcr_cassettes/appointments/VAOS_v2/get_appointment_200_partial_error.yml

```diff
@@ -0,0 +1,110 @@
+      Content-Type:
+      - application/json
+      User-Agent:
+      - Vets.gov Agent
+      Referer:
+      - https://review-instance.va.gov
+      X-Vamf-Jwt:
+      - eyJhbGciOiJSUzUxMiJ9.eyJhdXRoZW50aWNhdGVkIjp0cnVlLCJsYXN0TmFtZSI6Ik1vcmdhbiIsInN1YiI6IjEwMTI4NDYwNDNWNTc2MzQxIiwiYXV0aGVudGljYXRpb25BdXRob3JpdHkiOiJnb3YudmEudmFvcyIsImdlbmRlciI6IkZFTUFMRSIsImlzcyI6Imdvdi52YS52YW1mLnVzZXJzZXJ2aWNlLnYyIiwidmFtZi5hdXRoLnJlc291cmNlcyI6WyJeLiooXC8pP3BhdGllbnRbc10_XC8oSUNOXC8pPzEwMTI4NDYwNDNWNTc2MzQxKFwvLiopPyQiLCJeLiooXC8pP3NpdGVbc10_XC8oZGZuLSk_OTgzXC9wYXRpZW50W3NdP1wvNzIxNjY4NVwvYXBwb2ludG1lbnRzKFwvLiopPyQiLCJeLiooXC8pP3NpdGVbc10_XC8oZGZuLSk_NjY4XC9wYXRpZW50W3NdP1wvMTYxNzM3XC9hcHBvaW50bWVudHMoXC8uKik_JCIsIl4uKihcLyk_c2l0ZVtzXT9cLyhkZm4tKT85ODRcL3BhdGllbnRbc10_XC81NTIxNjEwNDRcL2FwcG9pbnRtZW50cyhcLy4qKT8kIl0sInNzbiI6Ijc5NjA2MTk3NiIsInNzdCI6MTYzOTQzODEwNSwicGF0aWVudCI6eyJmaXJzdE5hbWUiOiJKYWNxdWVsaW5lIiwibGFzdE5hbWUiOiJNb3JnYW4iLCJnZW5kZXIiOiJGRU1BTEUiLCJpY24iOiIxMDEyODQ2MDQzVjU3NjM0MSIsImRvYiI6IjE5NjItMDItMDciLCJkYXRlT2ZCaXJ0aCI6IjE5NjItMDItMDciLCJzc24iOiI3OTYwNjE5NzYifSwidmFtZi5hdXRoLnJvbGVzIjpbInZldGVyYW4iXSwicmlnaHRPZkFjY2Vzc0FjY2VwdGVkIjp0cnVlLCJleHAiOjE2Mzk0MzkwMDYsImp0aSI6IjVjNmFmYzUyLWJkZDctNGVhMi04OTM0LTI5NTk2YWUyZjUzMCIsImlkVHlwZSI6IklDTiIsImRhdGVPZkJpcnRoIjoiMTk2MjAyMDciLCJ2ZXJzaW9uIjoyLjYsImVkaXBpZCI6IjEwMTM1OTk3MzAiLCJ2aXN0YUlkcyI6W3sicGF0aWVudElkIjoiNzIxNjY4NSIsInNpdGVJZCI6Ijk4MyJ9LHsicGF0aWVudElkIjoiMTYxNzM3Iiwic2l0ZUlkIjoiNjY4In0seyJwYXRpZW50SWQiOiI1NTIxNjEwNDQiLCJzaXRlSWQiOiI5ODQifV0sImZpcnN0TmFtZSI6IkphY3F1ZWxpbmUiLCJzdGFmZkRpc2NsYWltZXJBY2NlcHRlZCI6dHJ1ZSwibmJmIjoxNjM5NDM3OTI2LCJkb2IiOiIxOTYyMDIwNyIsImxvYSI6Mn0.rrz0bmrUqXf9NZZtShsP0Ub5ZRiI2LUqgqncs-M9rtBTUyDyW8qUBO7MhYvmJEhA40S66eOh0-JZyDJYKUVg4Hl-1ZRiVc5uX-0I1MhxoQ0DYR886bH373Eybuh3AU30gyohkBun0M3NQV2C3bxs-f0_wkAuyNIR-Rxm6nysei5ZmnQvzJChHg1YKtQKNUUUIVOPDCro_q6pTw_QQA7ZdIVxmT4f18xHODxs9aUXnHOFJPzUK32VbPQVtVYRQWK_SFisG42-Rhxg3O_rDiUeFSxfV-Uhjo2RBh9W0utGuMW9qDbRx1AqQrIsbBM7YeaNhno-BBsfQyeX3--GlXq1YA
+      X-Request-Id:
+      - ''
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Date:
+      - Mon, 13 Dec 2021 23:28:26 GMT
+      Content-Type:
+      - application/json
+      Content-Length:
+      - '653'
+      Server:
+      - openresty
+      X-Vamf-Version:
+      - 1.14.1
+      B3:
+      - 8c17e441da8d346f6934d442b0a0ac7c-e1581452019f7fc4-0
+      Access-Control-Allow-Headers:
+      - x-vamf-jwt
+      X-Vamf-Build:
+      - 20a82ca
+      X-Vamf-Timestamp:
+      - '2021-11-26T16:29:10+0000'
+      Access-Control-Allow-Origin:
+      - "*"
+      Access-Control-Allow-Methods:
+      - GET,OPTIONS
+      Access-Control-Max-Age:
+      - '3600'
+      X-Envoy-Upstream-Service-Time:
+      - '424'
+      Strict-Transport-Security:
+      - max-age=63072000; includeSubDomains; preload
+    body:
+      encoding: UTF-8
+      string: '{"failures":[
```

Comment:
It doesn't truly impact anything, but I'd suggest adding failures to the end of this rather than the beginning. That's what will happen in real life and it's best to keep these cassettes as close to real life as we can. The formatting is also weird, but I see that's the case in the VCR you copied. But it might be nice to clean it up a little.

Reply:
changed